clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/PatchExecutor.scala

```diff
@@ -20,6 +20,8 @@ class PatchExecutor[CI <: ClioIndex](patchCommand: PatchCommand[CI]) extends Exe
 
   private lazy val queryAllJson: Json = Json.obj()
 
+  private lazy val parallelism = 32
```

Comment:
Does this help?
Why 32?
How do parallel requests to the server keep the client from timing out?
More smaller upserts?

Reply:
It means the client isn't waiting around as long while the server thinks. It succeeded when run against prod.

Reply:
Should put a scaladoc explaining that it's pretty much an arbitrary choice.

Reply:
OK. It works if it works. The comment could include the reason too.